docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/stripe.md

```diff
@@ -43,7 +43,9 @@ However, our team only tested the following payment methods:
 
 ## Current limitations
 
-The Stripe App has limited or no support for multi-capture. Partial capture of payment for orders with multiple items isn't covered. So, payments can’t be partially canceled. One payment intent is created per order, and the payment for the order can either be authorized, captured, or cancelled from Stripe's perespective.
+1. The Stripe App has limited or no support for multi-capture. Partial capture of payment for orders with multiple items isn't covered. So, payments can’t be partially canceled. One payment intent is created per order, and the payment for the order can either be authorized, captured, or cancelled from Stripe's perespective.
+2. Payments can be properly canceled only from the the Back Office and not from the Stripe Dashboard. 
```

Comment:
Explanation: cancellation only works from Spryker to Stripe and not the other way around